tests/projects/test_project.py

```diff
@@ -2314,6 +2315,27 @@ def test_workflow_path_with_project_workdir():
     assert path == "./context/./workdir/workflow.py"
 
 
+@pytest.mark.parametrize(
+    "alert_name_in_config, alert_name_as_func_param",
+    [
+        (None, None),
+        (None, ""),
+        ("", None),
+        ("", ""),
+    ],
```

Comment:
Maybe add a case where alert_name_in_config or  alert_name_as_func_param  is provided and no err is raised?

Reply:
@yaelgen It has been moved to an integration test (as described in the PR description), and a test for that already exists.